client/src/components/header/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { useContext } from 'react';
+import { Link, NavLink, withRouter } from 'react-router-dom';
+import './style.css';
+import { Modal, Menu, Dropdown } from 'antd';
+import { FaBookmark, FaUserAlt, AiFillHome, IoMdLogOut } from 'react-icons/all';
+import Logo from '../../assets/Logo.svg';
+import { AuthContext } from '../../container/authContext';
+
+const Header = ({ history }) => {
+  const { userInfo, isAuth, showLoginModal, signOut } = useContext(AuthContext);
+  // console.log(userInfo);
+
+  const info = () => {
+    Modal.info({
+      title: 'Course Search',
+      content: (
+        <div>
+          <p>
+            {' '}
+            The place where you will find the best courses on the internet. With
+            our app we want to help the people through searching among different
+            platform about their favorite courses, by one click you will find
+            all the top courses.
+          </p>
+        </div>
+      ),
+      onOk() {},
+    });
+  };
+  const menu = (
+    <Menu>
+      <Menu.Item key="1" onClick={() => history.push('/')}>
+        <AiFillHome />
+        Home
+      </Menu.Item>
+
+      {isAuth && (
+        <Menu.Item key="2" onClick={() => history.push('/FavoritePage')}>
+          <FaBookmark />
+          Favorite
+        </Menu.Item>
+      )}
+      {isAuth && (
+        <Menu.Item key="3" onClick={signOut}>
+          <IoMdLogOut />
+          logout
+        </Menu.Item>
+      )}
+    </Menu>
+  );
+  return (
+    <div className="container">
```

Comment:
you don't need this div here because ```<div className="header-container">``` wrap everything so you can apply your style on it